internal/weldrcheck/utils.go

```diff
@@ -64,3 +69,26 @@ func setupTestState(socketPath string, timeout time.Duration) (*TestState, error
 
 	return &state, nil
 }
+
+// Create a temporary repository
+func setUpTemporaryRepository() (string, error) {
+	dir, err := ioutil.TempDir("/tmp", "osbuild-composer-test-")
+	if err != nil {
+		return "", err
+	}
+	cmd := exec.Command("createrepo_c", path.Join(dir))
+	err = cmd.Start()
+	if err != nil {
+		return "", err
+	}
+	err = cmd.Wait()
+	if err != nil {
+		return "", err
+	}
+	return dir, nil
+}
+
+// Remove the temporary repository
+func tearDownTemporaryRepository(dir string) error {
+	return os.RemoveAll(dir)
+}
```

Comment:
Same functions in `osbuild-dnf-json-tests/main_test.go`. Makes sense to have them in utils.go and then reuse them.

Reply:
Right, I was going to look for a good common location for this. internal/test/helpers.go might be the right place.